sklearn/feature_selection/univariate_selection.py

```diff
@@ -596,7 +596,9 @@ def _get_support_mask(self):
 
         n_features = len(self.pvalues_)
         sv = np.sort(self.pvalues_)
-        selected = sv[sv <= float(self.alpha) / n_features * (np.arange(n_features) + 1)]
+        selected = sv[sv <= float(self.alpha) / n_features
+                    * (np.arange(n_features) + 1)]
+        print(selected)
```

Comment:
Oops.. that was in preparation for testing and fixing  test-case.. never needed and I forgot.. fixed now.